R/i_check_column_name.R

```diff
@@ -0,0 +1,22 @@
+i_check_column_name <- function(x, the_columns) {
```

Comment:
Since this returns a scalar boolean, `include_column` might be a better way of naming it. Thibaut wanted the convention of naming internal functions starting with `i_` so that it's easier to find them in a search.